indico/modules/events/abstracts/models/reviews.py

```diff
@@ -196,3 +207,6 @@ def can_view(self, user):
         elif user == self.user:
             return True
         return self.abstract.can_judge(user) or self.abstract.can_convene(user)
+
+    def get_url_for(self):
+        return url_for('.review_abstract', self.abstract, self.track)
```

Comment:
I'd use the full name, not blueprint-relative. Blueprint-relative endpoint references in a model are somewhat dirty and fragile